packages/next/client/dev/error-overlay/hot-dev-client.js

```diff
@@ -87,6 +88,7 @@ function clearOutdatedErrors() {
 // Successful compilation.
 function handleSuccess() {
   clearOutdatedErrors()
+  clearFullRefreshStorage()
```

Comment:
We should only show this once per session and not clear it after the next compilation as having to click reload for each change could be tedious.

Reply:
I‘ll remove this line then 🙂